chainstate/src/detail/tests/test_framework/framework.rs

```diff
@@ -28,7 +28,7 @@ use crate::{
     detail::{
         tests::{
             test_framework::{BlockBuilder, TestFrameworkBuilder, TransactionBuilder},
-            TestBlockInfo,
+            TestBlockInfo, *,
```

Comment:
This doesn't make sense. This is like saying "waiter, please get me the chicken and caeser salad, and get me the whole menu". Well, if you're ordering the whole menu, you don't need to specifically order the chicken... it's part of the menu.

Reply:
It's understandable. This does not remove responsibility from myself, but weird why Clippy don't mark it as an issue. 

Anyway, I itemized imports and removed `*`.